packages/wp-story-editor/src/components/metaBoxes/menuItem/index.js

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { __ } from '@web-stories-wp/i18n';
+import { useCallback } from '@web-stories-wp/react';
+import { trackEvent } from '@web-stories-wp/tracking';
+import { Tooltip } from '@web-stories-wp/story-editor';
+import {
+  Button,
+  BUTTON_SIZES,
+  BUTTON_TYPES,
+  BUTTON_VARIANTS,
+  Icons,
+  PLACEMENT,
+} from '@web-stories-wp/design-system';
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import useMetaBoxes from '../useMetaBoxes';
+
+const Box = styled.div`
+  width: 32px;
+  height: 32px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+const Space = styled.span`
+  width: 8px;
+`;
+
+const Item = styled.div`
+  order: -1;
+`;
```

Comment:
Hmm this really messes with the tab order when navigating with the keyboard. The DOM structure is now slightly different too, causing a slight layout:

Before:

![Screenshot 2021-09-22 at 10 14 20](https://user-images.githubusercontent.com/841956/134307652-9f010843-0a29-4159-9f76-79dd0801c7e5.png)

After:

![Screenshot 2021-09-22 at 10 14 30](https://user-images.githubusercontent.com/841956/134307650-68ebaecd-1fea-40ba-9c95-580006cf7c83.png)


See regression on Percy: https://percy.io/google/web-stories-wp-e2e/builds/12843630/changed/725855111

See how order of elements changed in the DOM:

Before:

![Screenshot 2021-09-22 at 10 18 06](https://user-images.githubusercontent.com/841956/134308071-d354dfc1-8d8e-4d10-931d-d22dc914e4e7.png)

After:

![Screenshot 2021-09-22 at 10 18 37](https://user-images.githubusercontent.com/841956/134308099-afa374ce-1562-4fa2-b0c6-51671f9930d5.png)

Reply:
In that case, without Slot/Fill, we will have to keep a `<AdditionalMenuItems />` area inside `<MenuItems>` so the menu items can be inserted in the same order.

Now

![single-item-menu-order](https://user-images.githubusercontent.com/6297436/134338529-7fa11741-e0d4-4a89-bfeb-5b71c9ded9b6.gif)


If required In the future more menu items can also be inserted inside `<AdditionalMenuItems />` and the tab should still work

![order](https://user-images.githubusercontent.com/6297436/134338664-e5b402ce-a4b5-4ae7-8ecf-544148380cd7.gif)

Reply:
Done in https://github.com/google/web-stories-wp/pull/9027/commits/784b712c99125b55847152bb2412953b8011000f 

Does it make sense?

Reply:
Before

![before](https://user-images.githubusercontent.com/6297436/134339865-c32454ca-3f76-4d30-8c55-193a54b683f8.png)



After:

![after](https://user-images.githubusercontent.com/6297436/134339884-0974f910-ba8c-4743-aaf5-0b55e17eb3ac.png)